extensions/3rdparty/RSS/magpierss/rss_parse.inc

```diff
@@ -147,10 +147,10 @@ class MagpieRSS {
         $el = $element = strtolower($element);
         $attrs = array_change_key_case($attrs, CASE_LOWER);
 
-        // check for a namespace, and split if found
+        // check for a namespace, and explode if found
         $ns = false;
         if ( strpos( $element, ':' ) ) {
-            list($ns, $el) = split( ':', $element, 2);
+            list($ns, $el) = explode( ':', $element, 2);
         }
         if ( $ns and $ns != 'rdf' ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace "and" with "&&". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2010)